Error occurred at 12-30-14-16:23:41 
OperationalError
thermDaemon.py
302

